sdmetrics/reports/multi_table/_properties/relationship.py

```diff
@@ -0,0 +1,76 @@
+from sdmetrics.multi_table.statistical import CardinalityShapeSimilarity
+from sdmetrics.reports.multi_table._properties.base import BaseMultiTableProperty
+from sdmetrics.reports.multi_table.plot_utils import get_table_relationships_plot
+
+
+class CardinalityShapeSimilarityProperty(BaseMultiTableProperty):
+    """``CardinalityShapeSimilarityProperty`` class.
+
+    Property that uses ``sdmetrics.multi_table.statistical.CardinalityShapeSimilarity`` metric
+    in order to compute and plot the scores of cardinality shape similarity in the given tables.
+    """
+
+    def __init__(self):
+        self._metric_results = {}
+
+    def get_score(self, real_data, synthetic_data, metadata, progress_bar=None):
+        """Get the average score of cardinality shape similarity in the given tables.
+
+        Args:
+            real_data (pandas.DataFrame):
+                The real data.
+            synthetic_data (pandas.DataFrame):
+                The synthetic data.
+            metadata (dict):
+                The metadata, which contains each column's data type as well as relationships.
+            progress_bar (tqdm.tqdm or None):
+                The progress bar object. Defaults to ``None``.
+
+        Returns:
+            float:
+                The average score for the property for all the individual metric scores computed.
+        """
+        for relation in metadata.get('relationships', []):
+            relationships_metadata = {'relationships': [relation]}
+            try:
+                self._metric_results = CardinalityShapeSimilarity.compute_breakdown(
+                    real_data,
+                    synthetic_data,
+                    relationships_metadata
+                )
+            except Exception:
+                pass
```

Comment:
I think in the other metrics we are appending the error message to a new column with the header `Error`. We should do the same. That is the old behavior too I believe